database/seed.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-console */
+/* eslint-disable no-underscore-dangle */
+const _ = require('underscore');
```

Comment:
Would consider refactoring without use of underscore. Possible options for range include vanilla JS or faker.